graphene/types/tests/test_schema.py

```diff
@@ -69,115 +69,3 @@ def test_schema_requires_query_type():
     assert len(result.errors) == 1
     error = result.errors[0]
     assert error.message == "Query root type must be provided."
-
-
-class TestUnforgivingExecutionContext:
```

Comment:
The entire test suite is deleted...

If the execution context is safe to delete, then you should be able to just remove `execution_context_class=UnforgivingExecutionContext,` from each of these and they should still be passing.

Reply:
I think that this part should be handled by graphql-core since it is that library that is providing the ExecutionContext, not us.